### PR TITLE
Update README before changing repo name to GATK

### PR DESCRIPTION
**Pull request by vdauwera**
Added links to current GATK website + repos to stave off possible confusion as mentioned in #945

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,11 +3,17 @@
...
-Hellbender
-================
+GATK 4 (codename Hellbender)
+============================
 
 The public parts of the next generation of GATK/Picard methods engine and tools.
 
+This project is in a pre-alpha development stage and is not yet ready for general use. 
+
+If you are looking for the current version of GATK to use in production work, please see the [GATK website](http://www.broadinstitute.org), where you can download a precompiled executable, read documentation, ask questions and receive technical support.
```
Comment (droazen) [inline README.md]: Erm, the GATK website is not at www.broadinstitute.org :)

File: README.md [hunk 2]
```diff
@@ -3,11 +3,17 @@
...
+
+If you are looking for the codebase of the current production version of GATK, please see either the [GATK development framework repository](https://github.com/broadgsa/gatk-protected/) or the [full GATK tools repository](https://github.com/broadgsa/gatk-protected/).
```
Comment (droazen) [inline README.md]: Both links point to the same place -- the first one should be `https://github.com/broadgsa/gatk`

**Comment by droazen** [comment]
Back to @vdauwera for some patches

**Comment by vdauwera** [comment]
Can't seem to do git clone https://github.com/broadinstitute/hellbender/

This is my console output:

````
wmd16-c9e:codespace vdauwera$ git clone http://github.com/broadinstitute/hellbender/
Cloning into 'hellbender'...
remote: Counting objects: 22221, done.
remote: Compressing objects: 100% (142/142), done.
remote: Total 22221 (delta 47), reused 4 (delta 4), pack-reused 22046
Receiving objects: 100% (22221/22221), 36.63 MiB | 3.58 MiB/s, done.
Resolving deltas: 100% (9903/9903), done.
Checking connectivity... done.
Downloading src/test/resources/large/CEUTrio.HiSeq.WGS.b37.NA12878.20.21.bam (76.16 MB)
Username for 'http://github.com': vdauwera
Password for 'http://vdauwera@github.com': 
Error accessing media: src/test/resources/large/CEUTrio.HiSeq.WGS.b37.NA12878.20.21.bam (6b1304800e60c0ac0358df137bdad48b7857a36465b04fef3fbbb09380f04746)

Errors logged to /Users/vdauwera/codespace/hellbender/.git/lfs/objects/logs/20151005T220016.510795175.log.
Use `git lfs logs last` to view the log.
Downloading src/test/resources/large/CEUTrio.HiSeq.WGS.b37.NA12878.20.21.bam.bai (11.25 KB)
Username for 'http://github.com': 
````

Looks like I'm failing to download large test files. Do I need to be on VPN for this to work? Or is it expected and I should ignore it?

**Comment by vdauwera** [comment]
Well, looks like I got the code at least, if not the test data. Commits squashed and ready for re-review, @droazen ;)

**Pull request merged**